commit 506899deb3fff632034ce71b968f04f1ef7e8338
Author: Michael Cyr <mikecyr@us.ibm.com>
Date:   Fri Mar 17 00:48:21 2017 +0000

    scsi: ibmvscsis: Rearrange functions for future patches
    
    [ Upstream commit 79fac9c9b74f4951c9ce82b22e714bcc34ae4a56 ]
    
    This patch reorders functions in a manner necessary for a follow-on
    patch.  It also makes some minor styling changes (mostly removing extra
    spaces) and fixes some typos.
    
    There are no code changes in this patch, with one exception: due to the
    reordering of the functions, I needed to explicitly declare a function
    at the top of the file.  However, this will be removed in the next patch,
    since the code requiring the predeclaration will be removed.
    
    Signed-off-by: Michael Cyr <mikecyr@us.ibm.com>
    Signed-off-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Tested-by: Steven Royer <seroyer@linux.vnet.ibm.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1419_506899deb3fff632034ce71b968f04f1ef7e8338.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1419_506899deb3fff632034ce71b968f04f1ef7e8338.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
